library/cephadm_bootstrap.py

```diff
@@ -158,7 +158,7 @@ def run_module() -> None:
     backward_compat = dict(
         dashboard=dict(type='bool', required=False, remove_in_version='4.0.0'),
         firewalld=dict(type='bool', required=False, remove_in_version='4.0.0'),
-        monitoring=dict(type='bool', required=False, remove_in_version='4.0.0'),
+        monitoring=dict(type='bool', required=False, remove_in_version='4.0.0'),  # noqa: E501
```

Comment:
any reason for not addressing that 'line too long' error ?
```suggestion
        monitoring=dict(type='bool',
                        required=False,
                        remove_in_version='4.0.0'),
```